components/event-sources/reconciler/object/service.go

```diff
@@ -77,20 +77,19 @@ func WithMinScale(replicas int) ObjectOption {
 	}
 }
 
-// WithSpecTemplateLabel sets the value of an API object's metadata label
-func WithSpecTemplateLabel(key, val string) ObjectOption {
+// WithPodLabel sets the value of an API object's metadata label
```

Comment:
The comment doesn't really match anymore. How about something along the lines of "sets a label on a Service's template".